experimental/cloudstrapper/README.md

```diff
@@ -368,9 +368,9 @@
         - awsOrc8rRegion : Determines which Region hosts the Orc8r instance to be deleted
         - orc8rClusterName : Local folder with terraform state [ex: ~/magma-experimental/<Name of Cluster>]
 
-  - ansible-playbook -e "dirLocalInventory=<Local Dir>" cleanup.yaml  [ --tags *various* ]
+  - ansible-playbook -e "dirLocalInventory=<Local Dir>" -e "{"deleteStacks": [stackName1, stackName2]}" cleanup.yaml  [ --tags *various* ]
 
-  Available tags include: agw,eks,asg,es,rds,efs,natgw,igw,subnet,secgroup,vpc
+  Available tags include: agw,eks,asg,es,rds,efs,natgw,igw,subnet,secgroup,vpc,orc8r,keys
 
```

Comment:
What does the orc8r tag do?

Reply:
by default the clean playbook was getting rid of orc8r componenets, I just tagged that part to avoid the playbook to fail.

Reply:
This change LGTM. Cleanup inherently needs some more work, specifically for partial deployments, where it exits today if it can't find a certain resource. Ideal behavior should be to add a flag to see if the variable is defined and only then progress. We can connect on this.

Reply:
Now the cleanup playbook does not fails when the resources are not found and just continues.